devfile.yaml

```diff
@@ -1,14 +1,23 @@
-apiVersion: 1.0.0
 metadata:
   name: kubernetes-api
+projects:
+  - name: kubernetes-api
```

Comment:
Well, forking doesn't work well / is not easy anyway in GO (since it changes the whole path of the Go package / Go module). 
In addition, I'm not sure we have the choice here, at least for now, since we have to clone the project in `src/github.com/devfile/kubernetes-api` through the `clonePath` project field.
It is necessary to have the GO language server working (Che doesn't support using GO LS in module mode due to the limitationthat CheTheia always opens the workspace at `/projects`).

Reply:
At least it covers the case when contributors want to contribute on branches of this repository, and use PR checks.